.github/workflows/zip.yml

```diff
@@ -53,18 +53,22 @@ jobs:
           ref: ${{ steps.settings.outputs.branch }}
 
       # ------------------------------------------------------------------------------
-      # Setup Node.
```

Comment:
Fixing this is good/cool but:

1. It has nothing to do with the rest of the PR
2. aren't we supposed to be replacing TUT with PUP?

Reply:
Was required for something else that was broken.